rpki-validator/src/main/resources/application.properties

```diff
@@ -92,3 +92,8 @@ management.endpoints.enabled-by-default=false
 management.endpoint.prometheus.enabled=true
 management.endpoints.web.exposure.include=prometheus
 
+# Enabling strict validation will not allow stale manifest, stale crl, and reject manifest with entries referring to non-existent objects.
```

Comment:
Do stale manifests and crls also get rejected?

Reply:
Yes